src/Accordion.php

```diff
@@ -170,9 +185,12 @@ public function renderItem($header, $item, $index)
             $options = ArrayHelper::getValue($item, 'contentOptions', []);
             $options['id'] = $id;
             Html::addCssClass($options, ['widget' => 'collapse']);
-            if ($index === 0) {
-                Html::addCssClass($options, 'show');
+
+            // check if accordion expanded, if true add show class
+            if($this->_isAccordionExpanded($item)){
+                $options['class'] = ' show';
```

Comment:
Like this you override existing classes like [`collapse` class](https://github.com/yiisoft/yii2-bootstrap4/pull/196/files#diff-0450c68805ce55d68227126fef5003bbR187). Replace with `Html::addCssClass($options, 'show');` will preserve them.